source/object-model/computed-properties.md

```diff
@@ -26,20 +26,28 @@ var ironMan = Person.create({
 
 ironMan.get('fullName'); // "Tony Stark"
 ```
-Notice that the `fullName` function calls `property`. This declares the function to be a computed property, and the arguments tell Ember that it depends on the `firstName` and `lastName` attributes.
+
+This declares the function to be a computed property, and the arguments tell Ember that it depends on the `firstName` and `lastName` attributes.
 
 Whenever you access the `fullName` property, this function gets called, and it returns the value of the function, which simply calls `firstName` + `lastName`.
 
 #### Alternate invocation
 
-At this point, you might be wondering how you are able to call the `.property` function on a function.  This is possible because Ember extends the `function` prototype.  More information about extending native prototypes is available in the [disabling prototype extensions guide](../../configuring-ember/disabling-prototype-extensions/). If you'd like to replicate the declaration from above without using these extensions you could do so with the following:
+You might have seen computed properties written using the `.property` style
+syntax:
 
 ```javascript
-  fullName: Ember.computed('firstName', 'lastName', function() {
-    return this.get('firstName') + ' ' + this.get('lastName');
-  })
+fullName: function() {
+  return this.get('firstName') + ' ' + this.get('lastName');
+}.property('firstName', 'lastName')
 ```
 
+This is possible because Ember extends the `function` prototype. This is no
+longer the recommended syntax.
```

Comment:
I am very much not in favor of integrating caveats about prototype extensions into the main flow.

I'd like to see us pick a happy path and talk only about that happy path in the primary documentation. Anything that deviates from the happy path can be moved to an auxiliary guide. If we make non-proto-extension syntax the default, let's update to that everywhere and create a legacy appendix describing how that syntax works for folks who need to work on code bases that use prototype extension.